network/builder/src/builder.rs

```diff
@@ -327,23 +322,10 @@ impl NetworkBuilder {
             None
         };
 
-        // union pubkeys from addrs with pubkeys directly in config
-        let addr_pubkeys_iter = seed_addrs.iter().map(|(peer_id, addrs)| {
-            let pubkey_set: HashSet<_> = addrs
-                .iter()
-                .filter_map(NetworkAddress::find_noise_proto)
-                .collect();
-            (*peer_id, pubkey_set)
-        });
-        for (peer_id, pubkey_set) in addr_pubkeys_iter {
-            seed_pubkeys.entry(peer_id).or_default().extend(pubkey_set);
-        }
```

Comment:
maybe I missed something, but it looks like we're not merging the pubkeys from seed addrs anywhere without this?

Reply:
Now, by nature of creating a `TrustedPeer`, the addresses' pubkeys are merged with the pubkeys.  See: `TrustedPeer::new()`

Reply:
right, but that's only if it's created using new. serde's Deserialize doesn't call new iirc, so the pubkeys won't be merged after deserializing from the config

Reply:
Ah, yeah I see now.  I just put this code back now.